### PR TITLE
Showcase update

### LINKED ISSUE #136: Refresh the showcase workflows
The showcase workflows (showed on the welcome screen) still use only widgets present in v0.1. They should be updated to include the new widgets

Cell Landscape could use Lovain clustering
Markers could use Markers widget

### PR DESCRIPTION
**Pull request by BlazZupan**
##### Issue
Fixes #136

##### Description of changes
- renames showcase files (ShowCase -> ShowCase)
- replaces k-means with Louvain clustering
- replaces Datasets + Table combination with Gene Markers

##### Includes
- [ ] Code changes
- [ ] Tests
- [X] Documentation

### CODE REVIEW

File: orangecontrib/single_cell/launcher/config.py [hunk 1]
```diff
@@ -34,21 +34,21 @@ def image_filename(filename):
 
     items = [
         WelcomeScreenSpecs.Item(
-            path=workflow_filename("ShowCase-CellType.ows"),
+            path=workflow_filename("Showcase-CellType.ows"),
             icon=image_filename("CellTypeDiscovery.svg"),
             active_icon=image_filename("CellTypeDiscoveryColor.svg"),
             text="Cell Type Discovery",
             tooltip="Use k-means clustering and tSNE to observe cell types",
```
Comment (lanzagar) [inline orangecontrib/single_cell/launcher/config.py]: The tooltip should be updated too: k-means -> Louvain

File: orangecontrib/single_cell/tutorials/Showcase-CellType.ows [hunk 2]
```diff
@@ -0,0 +1,2300 @@
...
+	<links>
+		<link enabled="true" id="0" sink_channel="Data" sink_node_id="2" source_channel="Selected Data" source_node_id="0" />
+		<link enabled="true" id="1" sink_channel="Data" sink_node_id="1" source_channel="Data" source_node_id="3" />
+		<link enabled="true" id="2" sink_channel="Data" sink_node_id="4" source_channel="Data" source_node_id="3" />
+		<link enabled="true" id="3" sink_channel="Data" sink_node_id="0" source_channel="Data" source_node_id="4" />
+	</links>
+	<annotations>
+		<text font-family="Helvetica" font-size="14" id="0" rect="(45.0, 115.0, 150.0, 82.0)" type="text/plain">Load the data on bone marrow mononuclear cells with AML (Zheng et. al. 2017).  </text>
+		<text font-family="Helvetica" font-size="20" id="1" rect="(15.0, 116.0, 150.0, 32.0)" type="text/plain">(1)</text>
+		<text font-family="Helvetica" font-size="14" id="2" rect="(243.0, 351.0, 150.0, 82.0)" type="text/plain">See the data in a spreadsheet. Data comes with pre-normalized expressions values.</text>
```
Comment (lanzagar) [inline orangecontrib/single_cell/tutorials/Showcase-CellType.ows]: Workflow comment (2):
expressions values -> expression values

File: orangecontrib/single_cell/tutorials/Showcase-Markers.ows [hunk 3]
```diff
@@ -0,0 +1,367 @@
...
+		<node id="3" name="Marker Genes" position="(210.0, 303.0)" project_name="Orange3-SingleCell" qualified_name="orangecontrib.single_cell.widgets.owmarkergenes.OWMarkerGenes" title="Marker Genes" version="" />
+	</nodes>
+	<links>
+		<link enabled="true" id="0" sink_channel="Data" sink_node_id="1" source_channel="Data" source_node_id="0" />
+		<link enabled="true" id="1" sink_channel="Data" sink_node_id="0" source_channel="Data" source_node_id="2" />
+		<link enabled="true" id="2" sink_channel="Genes" sink_node_id="0" source_channel="Genes" source_node_id="3" />
+	</links>
+	<annotations>
+		<text font-family="Helvetica" font-size="20" id="0" rect="(22.0, 18.0, 150.0, 32.0)" type="text/plain">(1)</text>
+		<text font-family="Helvetica" font-size="14" id="1" rect="(52.0, 17.0, 150.0, 124.0)" type="text/plain">Load the data on bone marrow mononuclear cells with AML (Zheng et. al. 2017). Data come pre-normalized and contains a variety of cell types.</text>
```
Comment (lanzagar) [inline orangecontrib/single_cell/tutorials/Showcase-Markers.ows]: Workflow comment (1):
Singluar/plural of data should be consistent.
Maybe also change to: "Data is pre-normalized and contains..."

Comment (BlazZupan) [reply orangecontrib/single_cell/tutorials/Showcase-Markers.ows]: Thanks for comments, I hope I have fixed the typos in the new commit.

**Pull request merged**